docs/synthetics/using-the-api.md

```diff
@@ -0,0 +1,777 @@
+        "failedRunCountToAlert": 1,
+        "notificationsEnabled": true,
+        "useOnlyAlertRuleIntegrations": false
+    }
+}'
+```
+
+### Create a HTTP Monitor
+To create a HTTP Monitor, we would send a HTTP request as follows:
+```
+curl -L -X POST 'https://apps.sematext.com/synthetics-api/api/apps/17174/monitors/http' \
+-H 'Authorization: apiKey 9bddb0a6-xxxx-xxxx-xxxx-397d15806cfd' \
+-H 'Content-Type: application/json' \
+--data-raw '{
+    "name": "Example HTTP monitor name",
+    "interval": "1m",
+    "enabled": true,
+    "locations": [
+        1
+    ],
+    "url": "https://www.google.com/",
+    "method": "GET",
+    "conditions": [
+        {
+            "id": 1,
+            "type": "ERROR",
+            "operator": "=",
+            "value": "",
+            "enabled": true
+        },
+        {
+            "id": 2,
+            "type": "RESPONSE_CODE",
+            "operator": "=",
+            "value": "200",
+            "enabled": true
+        },
+        {
+            "id": 3,
+            "type": "METRIC",
+            "key": "synthetics.time.response",
+            "operator": "<",
+            "value": "20000",
+            "enabled": true
+        }
+    ],
+    "alertRule": {
+        "schedule": [
+            {
+                "day": "Monday",
+                "index": 2,
+                "label": "MON",
+                "intervals": [],
+                "type": "ACTIVE"
+            },
+            {
+                "day": "Tuesday",
+                "index": 3,
+                "label": "TUE",
+                "intervals": [],
+                "type": "ACTIVE"
+            },
+            {
+                "day": "Wednesday",
+                "index": 4,
+                "label": "WED",
+                "intervals": [],
+                "type": "ACTIVE"
+            },
+            {
+                "day": "Thursday",
+                "index": 5,
+                "label": "THU",
+                "intervals": [],
+                "type": "ACTIVE"
+            },
+            {
+                "day": "Friday",
+                "index": 6,
+                "label": "FRI",
+                "intervals": [],
+                "type": "ACTIVE"
+            },
+            {
+                "day": "Saturday",
+                "index": 7,
+                "label": "SAT",
+                "intervals": [],
+                "type": "ACTIVE"
+            },
+            {
+                "day": "Sunday",
+                "index": 1,
+                "label": "SUN",
+                "intervals": [],
+                "type": "ACTIVE"
+            }
+        ],
+        "minDelayBetweenNotificationsInMinutes": "10",
+        "backToNormalNeeded": true,
+        "failedRunCountToAlert": 1,
+        "notificationsEnabled": true,
+        "useOnlyAlertRuleIntegrations": false
+    },
+    "monitorSSLExpiry": true,
+    "monitorSSLChange": true,
+    "allowInsecureSSL": false
+}'
+```
+## Get all Monitors for an app
```

Comment:
app should always be capitalized when used in text.

Reply:
Yes it should, I will capitalize.